src/twisted/web/test/test_http_headers.py

```diff
@@ -371,6 +371,9 @@ def test_getRawHeadersDefaultValue(self):
         h = Headers()
         default = object()
         self.assertIdentical(h.getRawHeaders(u"test", default), default)
+        self.assertIdentical(h.getRawHeaders(u"test", None), None)
+        self.assertEqual(h.getRawHeaders(u"test", [None]), [None])
+        self.assertEqual(h.getRawHeaders(u"test", [u"\N{SNOWMAN}"]), [u"\N{SNOWMAN}"])
```

Comment:
I am not aware of any project policy ... so for now is up to the branch author and the reviewer.
I made only a minor comment and I am OK to merge this as it is as long as you don't consider that this could improve the current code.

Project policies should be discussed over the mailing list, so feel free to merge this as it is.

Thanks!